frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -1408,13 +1408,13 @@ function ReaderFooter:addToMainMenu(menu_items)
                 text_func = function()
                     local prefix_text = ""
                     if self.settings.item_prefix == "icons" then
-                        prefix_text = _("Icons")
+                        prefix_text = C_("Status bar", "Icon")
```

Comment:
It seems to be? I think icon/letter reads a tad weird.

Reply:
Do you want to go for "Icons" and "Letters" in the plural instead?

Reply:
Oh, hah. Well, I mainly just meant to get rid of the word prefix. What's your preference?

Reply:
I like the plurals a tiny bit more